src/ansys/dpf/post/fluid_simulation.py

```diff
@@ -276,6 +281,45 @@ def _get_result(
                 "are mutually exclusive."
             )
 
+        # Raise for integrated result queried on anything else than its native location
+        if integrated:
+            if (native_location == "Faces" and location != locations.faces) or (
+                native_location == "Elements" and location != locations.elemental
```

Comment:
why are we not using the locations strings in here?

Reply:
@cbellot000 because they are not the native_location strings in the available results are not the same as what you can find in `locations`